chromoscores/scorefunctions.py

```diff
@@ -91,6 +143,19 @@ def peak_score(
     background_width,
     pseudo_count,
 ):
+    """
```

Comment:
should peak_score raise an error if peak_width//2 + background width > len(snippet)/2 ? or in some other scenarios ?